marklogic-client-api/src/main/java/com/marklogic/client/expression/PlanBuilder.java

```diff
@@ -1688,6 +1690,16 @@ public interface ModifyPlan extends PreparePlan, PlanBuilderBase.ModifyPlanBase
   * @return  a ModifyPlan object
   */
   public abstract ModifyPlan whereDistinct();
+public abstract ModifyPlan write();
+public abstract ModifyPlan write(PlanDocColsIdentifier docCols);
+public abstract ModifyPlan unnestInner(String inputColumn, String valueColumn);
+public abstract ModifyPlan unnestInner(PlanExprCol inputColumn, PlanExprCol valueColumn);
+public abstract ModifyPlan unnestInner(String inputColumn, String valueColumn, String ordinalColumn);
+public abstract ModifyPlan unnestInner(PlanExprCol inputColumn, PlanExprCol valueColumn, PlanExprCol ordinalColumn);
+public abstract ModifyPlan unnestLeftOuter(String inputColumn, String valueColumn);
+public abstract ModifyPlan unnestLeftOuter(PlanExprCol inputColumn, PlanExprCol valueColumn);
+public abstract ModifyPlan unnestLeftOuter(String inputColumn, String valueColumn, String ordinalColumn);
+public abstract ModifyPlan unnestLeftOuter(PlanExprCol inputColumn, PlanExprCol valueColumn, PlanExprCol ordinalColumn);
```

Comment:
Just for awareness, it will be necessary to regenerate this interface after the reference documentation for these server functions is added.  Regeneration will add javadoc for these methods.